src/main/java/mate/academy/springboot/swagger/dto/ProductRequestDto.java

```diff
@@ -0,0 +1,16 @@
+package mate.academy.springboot.swagger.dto;
+
+import java.math.BigDecimal;
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.Size;
+import lombok.Data;
+
+@Data
+public class ProductRequestDto {
+    @NotBlank
+    @Size(min = 1, max = 120)
+    private String title;
+    @NotBlank
```

Comment:
Could we use this annotation here? [See here](https://jakarta.ee/specifications/bean-validation/3.0/apidocs/index.html?jakarta/validation/constraints/NotBlank.html)